src/Header.vue

```diff
@@ -1,18 +1,19 @@
 <template>
   <div class="sc-header" :style="{background: colors.header.bg, color: colors.header.text}">
-    <img class="sc-header--img" :src="imageUrl" alt="" />
+    <img class="sc-header--img" :src="imageUrl" alt="" v-if="imageUrl" />
     <div class="sc-header--team-name"> {{teamName}} </div>
     <div class="sc-header--close-button" @click="onClose">
       <img src="./assets/close-icon.png" alt="" />
     </div>
   </div>
 </template>
 <script>
+
 export default {
   props: {
     imageUrl: {
       type: String,
-      required: true
+      default: ''
```

Comment:
why the blank url?

Reply:
I made it optional. If it's blank, no image is shown. It was sort of necessary with backward compatibility, in case `agentProfiles` was used without `titleImageUrl`, but with the upcoming changes I revert this line.